models/public/device_support.md

```diff
@@ -31,6 +31,7 @@
 | efficientnet-b0-pytorch | YES | YES | YES |
 | efficientnet-v2-b0 | YES | YES |    |
 | efficientnet-v2-s | YES | YES |    |
+| erfnet | YES | YES |    |
```

Comment:
are you tested and guarantee that model works on GPU?

Reply:
I have modified it to   | erfnet | YES |     |     |